mailbox/memory/src/test/java/org/apache/james/mailbox/inmemory/manager/InMemoryIntegrationResources.java

```diff
@@ -113,6 +113,13 @@ public InMemoryMailboxManager createMailboxManager(GroupMembershipResolver group
         return createResources(groupMembershipResolver).mailboxManager;
     }
 
+    public InMemoryMailboxManager createMailboxManager(GroupMembershipResolver groupMembershipResolver, Set<PreDeletionHook> preDeletionHooks) {
```

Comment:
It starts to be messy. Couldn't we say that the method above calls this one with an empty Set?

Reply:
:+1:  and for your information we added a refactoring task, to try providing a builder for that...